### ISSUE TITLE
Information needed from API with ping

### ISSUE THREAD

**Issue by MansMeg** [rOpenGov/pxweb#174]
Get more information:
- All limits (now it seem to be differences)
- Citation information (how should data be cited)
- Base level and structure
- languages and versions
- encoding used by the API

**Comment by jxpx91** [comment]
Hi Måns, I'm using the API using a JS program, my question is about getting the metadata in json-stat response, I need to know the HEADING and STUB values of the px file. At this moment I'm getting it from the px format response, but it means two calls (px format and json-stat format). In another response you use get_pxweb_metadata usign R, so how can I get metadata in a simple way?

**Comment by MansMeg** [comment]
Hi! Im not sure if I understand your question, but if you do a call to the table but without a query, you will get a metadata json object in return. Although, not a json-stat since the meta-data dont contain any statistical data.

**Comment by jxpx91** [comment]
Yes, if I do a GET without a query I get a json but it only contains title and variables. I need more metadata.

This is how I'm working:
- I'm using Postman
- If I call a GET to my PX API, I'll receive the JSON that we're talking about.
- If I call a POST to my PX API with a query and **format json-stat**, I'll receive a json-stat with all the information that I need less the HEADING and STUB.
- If I call a POST to my PX API with a query and **format px**, I'll receive all the metadata that I need with the HEADING and STUB also all the values and data.

I know that I can work with px format, but we want with json-stat. So, there is a way to get all that metadata with all the data and values on json-stat?

I attach the examples
![2019-06-13_10h12_39](https://user-images.githubusercontent.com/17054032/59444743-fe885f00-8dc3-11e9-8ee4-a4b340bdeaa6.png)
![2019-06-13_10h13_12](https://user-images.githubusercontent.com/17054032/59444744-fe885f00-8dc3-11e9-94b4-fab9b52bb5b5.png)
![2019-06-13_10h13_46](https://user-images.githubusercontent.com/17054032/59444745-fe885f00-8dc3-11e9-8b96-31869ecbb945.png)

**Comment by MansMeg** [comment]
Hi,

I cannot tell exactly what is included in the different object. In the R pxweb package we use the JSON format. If the thing you are after is not included in neither the meta data object nor the json-stat table content object I guess it only exists in the px object and hence you need to get it twice.

You can also send an e-mail to the pxweb developers at Statistics Sweden.

/Måns

**Comment by jxpx91** [comment]
Hi Måns, 

Yes... It's what I was afraid for, call two get, but it seems to be the "correct" way. Thank you for all your help and attention. I'll send an email to pxweb developers and if they have a solution I'll put it here.

Regards

Juan Pablo

**Comment by MansMeg** [comment]
This has now been solved in the next version.
